scripts/variantstore/wdl/GvsSitesOnlyVCF.wdl

```diff
@@ -17,9 +17,37 @@ workflow GvsSitesOnlyVCF {
         String? service_account_json_path
         File? gatk_override
         File AnAcAf_annotations_template
+        File ancestry_file
```

Comment:
The plan was to rename the wdl when it gets split into the sites only and the vat creation wdl

it might make sense to do that as a ticket we tackle sooner rather than later